README.adoc

```diff
@@ -60,47 +62,51 @@ If you wish to use different Infinispan version than provided by this starter, p
        </dependency>
     </dependencies>
  </dependencyManagement>
+----
 
-```
-
-## Using Embedded mode
-
-The starters operate in Client/Server mode by default. Enabling Embedded mode requires adding `infinispan-core`
-to the classpath.
-
-Once you add infinispan's `spring-boot-starter` as a dependency to your project, a bean of type `EmbeddedCacheManager` can
-be `autowired` into your java configuration classes. Just simply add:
+== Running in Embedded Mode
 
-```java
+. Add `infinispan-spring-boot-starter-embedded` to your project's classpath to enable Embedded mode.
```

Comment:
@karesti hopefully this is correct. please let me know if it needs revising. cheers.